src/io/cscape_result_io.jl

```diff
@@ -342,25 +343,26 @@ function _create_inputs_dataframe(
     coral_dict = merge(settle_probs_kwargs, corals_deployed)
 
     return DataFrame(;
+        scenario_id=scenario_id,
         dhw_scenario=dhws,
         cyc_scenario=cyclones,
-        RCP=repeat([rcp], n_draws),
-        thermal_tol_lb=repeat([lb_t], n_draws),
-        thermal_tol_int1=repeat([int_t1], n_draws),
-        thermal_tol_int2=repeat([int_t2], n_draws),
-        thermal_tol_ub=repeat([ub_t], n_draws),
-        init_heat_tol_mean=repeat([init_heat_tol_mean], n_draws),
-        init_heat_tol_std=repeat([init_heat_tol_std], n_draws),
-        heritability1=repeat([heritability1], n_draws),
-        heritability2=repeat([heritability2], n_draws),
-        plasticity=repeat([plasticity], n_draws),
-        intervention_start=repeat([intervention_start], n_draws),
-        intervention_duration=repeat([duration], n_draws),
-        intervention_frequency=repeat([frequency], n_draws),
-        deployment_area=repeat([deployment_area], n_draws),
-        n_deployment_locations=repeat([n_dep_locations], n_draws),
-        enhancement_mean=repeat([enhancement_mean], n_draws),
-        enhancement_std=repeat([enhancement_std], n_draws),
+        RCP=fill(rcp, (n_draws,)),
+        thermal_tol_lb=fill(lb_t, (n_draws,)),
+        thermal_tol_int1=fill(int_t1, (n_draws,)),
+        thermal_tol_int2=fill(int_t2, (n_draws,)),
+        thermal_tol_ub=fill(ub_t, (n_draws,)),
+        init_heat_tol_mean=fill(init_heat_tol_mean, (n_draws,)),
+        init_heat_tol_std=fill(init_heat_tol_std, (n_draws,)),
+        heritability1=fill(heritability1, (n_draws,)),
+        heritability2=fill(heritability2, (n_draws,)),
+        plasticity=fill(plasticity, (n_draws,)),
+        intervention_start=fill(intervention_start, (n_draws,)),
+        intervention_duration=fill(duration, (n_draws,)),
+        intervention_frequency=fill(frequency, (n_draws,)),
+        deployment_area=fill(deployment_area, (n_draws,)),
+        n_deployment_locations=fill(n_dep_locations, (n_draws,)),
+        enhancement_mean=fill(enhancement_mean, (n_draws,)),
+        enhancement_std=fill(enhancement_std, (n_draws,)),
```

Comment:
Don't think you need the implicit tuple, you can just do:

`fill(value, number)`

Reply:
Incidentally, the current way, I think you're allocating to create the one element tuple